src/main/java/com/puppycrawl/tools/checkstyle/api/FileContents.java

```diff
@@ -324,13 +324,4 @@ private boolean hasIntersectionWithSingleLineComment(int startLineNo, int startC
         return Collections.unmodifiableMap(clangComments);
     }
 
-    /**
-     * Checks if the current file is a package-info.java file.
-     *
-     * @return true if the package file.
-     */
-    public boolean inPackageInfo() {
-        return getFileName().endsWith("package-info.java");
```

Comment:
ups, it is "api", we can not remove this method for some period.
Lets deprecate it and define when we will remove it.

Reply:
done